src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/cumulative-metrics-transition-guide.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Transition guide for OpenTelemetry cumulative sum metrics
+metaDescription: 'A transition guide about a change in how New Relic handles OpenTelemetry cumulative sum metrics, and how to update your queries and alerts for that change.'
+redirects: 
+---
+
+Our OTLP metric pipeline is being upgraded to better support how we handle [cumulative metrics](/docs/data-apis/understand-data/metric-data/cumulative-metrics). As part of this upgrade we're transitioning the underlying [metric data type](/docs/data-apis/understand-data/metric-data/metric-data-type/#metric-types) for [monotonic cumulative sums](https://opentelemetry.io/docs/reference/specification/metrics/data-model/#sums) from a `gauge` type to the `cumulativeCount` type. The `cumulativeCount` type allows you to query both the cumulative and delta value associated with reported data. 
+
+Starting March 14, 2023, new accounts will have access to this new `cumulativeCount` type, but if your account was sending cumulative monotonic sums prior to March 14th, you won't have access to that metric type until June 13. On that date, the use of our `gauge` metric type for cumulative sums will be deprecated and reach end-of-life (EoL), and your cumulative sums will be stored as the `cumulativeCount` type. 
+
+**This 90 day EoL period is to give you time to update queries and alerts that make use of the affected metrics.** The sections below will teach you: 
+
+* How to know if your account is impacted
+* How to prepare for the EoL
+* How to preview the `cumulativeCount` functionality 
```

Comment:
```suggestion
* How to opt-in to the `cumulativeCount` functionality prior to the EoL date
```
Goal is to make this sound less like an early preview and more like switching over to the new experience sooner.